app/components/task/holder.hbs

```diff
@@ -41,12 +41,27 @@
         >
           {{#each this.availabletaskStatusList as |taskStatus|}}
             {{#if (not-eq taskStatus.key this.TASK_KEYS.ALL)}}
-              <option
-                value={{taskStatus.key}}
-                selected={{eq taskStatus.key this.status}}
-              >
-                {{taskStatus.displayLabel}}
-              </option>
+              {{#if @dev}}
+                {{#if (not-eq taskStatus.key this.TASK_KEYS.COMPLETED)}}
```

Comment:
Line 43 and 45 checks are similar. Instead of nesting these checks should we consider moving these checks to .js file? 

Something like this? 
```javascript
get taskStatusList() {
   const statusToDisplay = [...this.availabletaskStatusList];
   // filter the list 
   return statusToDisplay;
}
```

Reply:
Done with changes can you please check my latest commit